lifeguards.java

```diff
@@ -0,0 +1,68 @@
+//package usaco2018january;
+//All ten tests work no errors
+//30 min
+//Difficulty:3/10
+//I thought it was pretty easy, I could've given the variables better names
+
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.StringTokenizer;
+
+public class lifeguards
+{
+
+	public static void main(String[] args) throws IOException
+	{
+		BufferedReader br = new BufferedReader(new FileReader("lifeguards.in"));
+		PrintWriter pw = new PrintWriter(new BufferedWriter(new FileWriter("lifeguards.out")));
+		//-----------------------------------------------------------------------------------------------------------------------------------------------------------------------
+		StringTokenizer st;
+		int N = Integer.parseInt(br.readLine());
+		int[][] schedule =new int [2][N];

```

Comment:
It is bad to use array when different index means differently. You need carry a decoding map in your head to read your code! Use a simple class so you can use explicit member names. If you want to use array -- as it is simpler -- then always add comments right above every usage. For example:
 `// 0: start, 1: end`